chainstate/monitor.go

```diff
@@ -203,11 +202,11 @@ func (m *Monitor) SetChainstateOptions(options *clientOptions) {
 }
 
 // Start open a socket to the service provider and monitorConfig transactions
-func (m *Monitor) Start(ctx context.Context, handler MonitorHandler, onStop func()) error {
+func (m *Monitor) Start(_ context.Context, handler MonitorHandler, onStop func()) error {
 	if m.client == nil {
 		handler.SetMonitor(m)
 		m.handler = handler
-		m.logger.Info(ctx, fmt.Sprintf("[MONITOR] Starting, connecting to server: %s", m.buxAgentURL))
+		m.logger.Info().Msgf("[MONITOR] Starting, connecting to server: %s", m.buxAgentURL)
```

Comment:
Is it necessary to add `[MONITOR]`? We will have `[INFO]` here

Reply:
I did not want to change anything in monitor because currently we do not use it and maybe we will remove it in the future but if you want I can change it